client/js/src/main/scala/zio/zmx/client/frontend/views/ChartView.scala

```diff
@@ -146,8 +155,54 @@ object ChartView {
         div(
           cls := "w-1/2 h-full p-3 ml-2",
           span(
-            cls := "text-2xl font-bold",
-            "Some Diagram info"
+            cls := "text-xl font-bold",
+            "Diagram Config"
+          ),
+          form(
+            cls := "text-2xl m-2",
+            onSubmit.preventDefault
+              .mapTo(zipVar.now()) --> (zip => {
+              println(s"HERE ${zip}")
+              // I need to generate a
+              // Command.UpdateDiagram(DiagramConfig.from?????)
```

Comment:
This is the expected API, but I am struggling to know from what I should generate the `DiagramConfig`. I was thinking of having sort of like a `.ref` like [React.js](https://reactjs.org/docs/refs-and-the-dom.html) does and storing the UUDI in the HTML using a TAG, to later retrieve it. But before doing that, I wanted to check with you if I am not overcomplicating things.

Reply:
The above could lead to the API being 
`Command.UpdateDiagram(DiagramConfig.fromHTMLNode)`